app/controllers/lettings_logs_controller.rb

```diff
@@ -124,6 +115,11 @@ def org_params
 
 private
 
+  def authenticate_scope!
+    codes_only_export = codes_only_export?(params)
+    head :unauthorized and return if codes_only_export && !current_user.support?
```

Comment:
```suggestion
    head :unauthorized and return if codes_only_export? && !current_user.support?
```
in tandem with my other suggestion, i feel the local variable is a bit redundant and overkill and can probably be removed

Reply:
certainly given params not needed to be passed in.